Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2_test.py

```diff
@@ -1083,3 +1088,400 @@ def test_update_alerts_command(args, expected_error, mcafeensmv2_client):
+                                            ({"domain_id": 777, "limit": 1}, [{'policyId': 'mock'}]),
+                                            ({"domain_id": 777, "limit": 1, "all_results": True},
+                                            [{'policyId': 'mock'}, {'policyId': 'mock'}])])
+def test_list_device_policy_command__with_different_arguments(mocker, input, output, mcafeensmv2_client):
+    """
+    Given:
+        - A domain_id.
+    When:
+        - nsm-list_device_policy_command command is executed with and without limit, with and without all_results.
+    Then:
+        - Confirm the output is as expected(number of results, and ID = 0 dose not raise an error).
+    """
+    from McAfeeNSMv2 import list_device_policy_command
+    mocker.patch.object(mcafeensmv2_client, 'list_device_policy_request',
+                        return_value={})
+    mocker.patch.object(McAfeeNSMv2, 'capitalize_key_first_letter', return_value=[{"policyId": "mock"}, {"policyId": "mock"}]),
+    res = list_device_policy_command(client=mcafeensmv2_client, args=input)
+    assert res.outputs == output
+
+
+def test_list_device_policy_command__with_missing_arguments(mocker, mcafeensmv2_client):
+    """
+    Given:
+        - No domain_id.
+    When:
+        - list_device_policy_command command is executed, with missing arguments.
+    Then:
+        - Confirm the output is as expected(error message).
+    """
+    from McAfeeNSMv2 import list_device_policy_command
+    mocker.patch.object(mcafeensmv2_client, 'list_device_policy_request',
+                        return_value={None})
+    with pytest.raises(DemistoException) as e:
+        list_device_policy_command(client=mcafeensmv2_client, args={})
+
+    assert e.value.message == "Please provide a domain_id."
+
+
+@ pytest.mark.parametrize('input, output', [({"domain_id": 0}, [{'deviceId': 'mock'}, {'deviceId': 'mock'}]),
+                                            ({"domain_id": 777, "limit": 1}, [{'deviceId': 'mock'}]),
+                                            ({"domain_id": 777, "limit": 1, "all_results": True},
+                                             [{'deviceId': 'mock'}, {'deviceId': 'mock'}])])
+def test_list_domain_device_command_with_diffrent_arguments(mocker, mcafeensmv2_client, input, output):
+    """
+    Given:
+        - A domain id.
+    When:
+        - nsm-list_domain_device_command command is executed, with and without limit and with and without all_results.
+    Then:
+        - Confirm the output is as expected(number of results, and ID = 0 dose not raise an error).
+    """
+    from McAfeeNSMv2 import list_domain_device_command
+    mocker.patch.object(mcafeensmv2_client, 'list_domain_device_request',
+                        return_value={})
+    mocker.patch.object(McAfeeNSMv2, 'capitalize_key_first_letter', return_value=[{"deviceId": "mock"}, {"deviceId": "mock"}])
```

Comment:
It isn't critical but I thimk that the mocked returned value of 'capitalize_key_first_letter' func, should be a dict with keys that has a capital first letter (it would just make more sense in my opinion). 
Relevant to all tests that mock 'capitalize_key_first_letter'.